src/box/error.cc

```diff
@@ -156,9 +148,15 @@ const char *rmean_error_strings[RMEAN_ERROR_LAST] = {
 
 /** Format client error with arguments in `ap` according to error format. */
 static void
-client_error_create(struct error *e, va_list ap)
+client_error_create(struct error *e, const char *fmt, va_list ap)
```

Comment:
This function isn't required anymore and could be inlined into the `ClientError` constructor.

Reply:
Done.